### ISSUE TITLE
0001_initial.py migration fails due to out of date dependency on django-taggit

### ISSUE THREAD

**Issue by tomassommar** [nephila/djangocms-blog#104]
When installing djangocms-blog on Django 1.7.4 trough pip, and subsequently running 
```
python manage.py migrate
```
migration 0001_initial.py fails with error message:
```
/migrations/0001_initial.py", line 97, in Migration
    ('tags', taggit_autosuggest.managers.TaggableManager(to='taggit.Tag', through='taggit.TaggedItem', blank=True, help_text='A comma-separated list of tags.', verbose_name='Tags')),
TypeError: __init__() got an unexpected keyword argument 'to'
```
This seems to be because an obsolete version of django-taggit is installed, where to is not a valid parameter to TaggableManager. When installing django-taggit >= 0.12 the migration works.

**Comment by jedie** [comment]
Confirm that is works with:
```
Django==1.7.4
django-cms==3.0.9
django-taggit==0.12.2
django-taggit-autosuggest==0.2.6
django-taggit-templatetags==0.2.5
-e git+git@github.com:jedie/djangocms-blog.git@ec6f798b341e731fd2fc1b0b890dd639769d65e0#egg=djangocms_blog-authors
```

So why not just update django-taggit?


Maybe change https://github.com/nephila/djangocms-blog/blob/develop/setup.py#L42 and add **>0.12** ?

**Comment by yakky** [comment]
@tomassommar @jedie I will force a versioned dependency on 0.12.2 version of taggit

**Comment by jedie** [comment]
:+1:

**Comment by yakky** [comment]
Fixed in #106
